### PR TITLE
Correction of check test for update queries

### PR DESCRIPTION
**Pull request by takapi327**
From [1763](https://github.com/tpolecat/doobie/issues/1763)

### CODE REVIEW

File: modules/example/src/main/scala/example/AnalysisTest.scala [hunk 1]
```diff
@@ -49,7 +49,7 @@ object AnalysisTest {
     """.query[PGcircle]
   }
 
-  def update(code: String, name: Int): Update0 =
+  def update(name: String, code: String): Update0 =
```
Comment (takapi327) [inline modules/example/src/main/scala/example/AnalysisTest.scala:52]: This was not consistent with the table definition itself, so it was corrected and used as a test query.

File: modules/example/src/test/scala-2/example/AnalysisTestSpecs2.scala [hunk 2]
```diff
@@ -22,6 +22,6 @@ class AnalysisTestSpecs2 extends Specification with IOChecker {
   // check(AnalysisTest.arrayTest2)
   check(AnalysisTest.pointTest)
   // check(AnalysisTest.pointTest2)
-  // check(AnalysisTest.update("foo", 42))
+  checkOutput(AnalysisTest.update("foo", "bkah"))
```
Comment (takapi327) [inline modules/example/src/test/scala-2/example/AnalysisTestSpecs2.scala:25]: I wanted to test on [IOCheckerCheck](https://github.com/tpolecat/doobie/blob/main/modules/specs2/src/test/scala/doobie/specs2/CheckerTests.scala#L30), but this one seemed to be set up to test only simple queries, so I am testing on the example project.

Comment (jatcwang) [reply modules/example/src/test/scala-2/example/AnalysisTestSpecs2.scala]: It's ok :) I think at some point we need to realign all the tests (scalatest/munit/weaver etc etc) so they test all the scenarios

**Comment by jatcwang** [comment]
Thank you! :100:

**Pull request merged**